rpc_jobs/rpc_aio.yml

```diff
@@ -169,6 +169,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
No, an exception will change the build result, without having to change it explicitly. The only reason to set it manually is so that it can be read.